talk/talk-api.yaml

```
@@ -1157,6 +1154,161 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ResponseError'
+  /chats:
+    get:
+      tags:
+        - Messages
+      summary: Retrieve a list of chats.
+      operationId: getChats
+      parameters:
+        - $ref: '#/components/parameters/QueryPageNumber'
+        - $ref: '#/components/parameters/QueryPageSize'
+        - $ref: '#/components/parameters/QuerySort'
+        - name: userNumber
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: The chats have been retrieved.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Chats'
+        default:
+          description: Bad request, security violation, or internal server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseError'
+    post:
+      tags:
+        - Messages
+      summary: Send message
```

```suggestion
      summary: Create a message in a chat (if present, or else create a chat also) and send the message
```
maybe something like that